client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -195,6 +198,18 @@ const onboarding: Flow = {
 				return [ `/home/${ providedDependencies.siteSlug }`, null ];
 			}
 
+			const playgroundId = getQueryArg( window.location.href, 'playground' );
+			if ( playgroundId && providedDependencies.siteSlug ) {
+				return [
+					addQueryArgs( withLocale( '/setup/site-setup/importerPlayground', locale ), {
```

Comment:
I don't think you need the locale since the user is logged in at this point.

Reply:
This must be used? [http://calypso.localhost:3000/setup/onboarding/playground](https://href.li/?http://calypso.localhost:3000/setup/onboarding/playground) can be accessed logged-in and not.